packages/client/src/graphql/Foretold__GraphQL__GetAgentMeasurements.re

```diff
@@ -0,0 +1,183 @@
+type measurable = {
+  id: string,
+  name: string,
+  state: Context.Primary.MeasurableState.t,
+  stateUpdatedAt: option(MomentRe.Moment.t),
+  expectedResolutionDate: option(MomentRe.Moment.t),
+};
+
+// @todo:
```

Comment:
what's the plan for this? To do in future PRs, or should we make an issue?

Reply:
Also, note that we have a ``connections`` type in ``Context__Primary``. Could you use that maybe?

Reply:
I have added description to each todos.  At this moment I do not properly understand how to use "Context.Primary.Connection" module in a right way, this is why I added todos. It need more time to think it over.

Reply:
Fixed.